src/server/system_services/tier_server.js

```diff
@@ -357,7 +357,11 @@ async function update_bucket_class(req) {
                 .map(obj => obj.tier.name);
             for (const [index, tier] of tiers.entries()) {
                 const old_tier = old_tiers[index];
-                const new_name = `${bucket.name.unwrap()}.${Math.round(Date.now() / 1000)}.${index}`;
+                let new_name = `${bucket.name.unwrap()}.${Math.round(Date.now() / 1000)}.${index}`;
```

Comment:
Cant we just remove the division by 1k? Any chance for 2 requests in the same millisec(more than a 1/100 random?) In anyway i think we should use the same format. So if you use this random, add it also to the initial new_name